scene/resources/immediate_mesh.h

```diff
@@ -75,6 +75,8 @@ class ImmediateMesh : public Mesh {
 	Vector<uint8_t> surface_vertex_create_cache;
 	Vector<uint8_t> surface_attribute_create_cache;
 
+	const Vector3 small_vec3 = Vector3(CMP_EPSILON, CMP_EPSILON, CMP_EPSILON);
```

Comment:
We usually name constants in `SCREAMING_SNAKE_CASE` (like class enums and macros).

Reply:
I was missing it since it is a fairly rare case for me to define a class constant. Fixed.